### PR TITLE
Add a new multi trigger example and explain use of "imaginary" options

### PR DESCRIPTION
**Pull request by alwaysintreble**

### CODE REVIEW

File: WebHostLib/static/assets/tutorial/archipelago/triggers_en.md [hunk 1]
```diff
@@ -66,4 +62,74 @@ For example:
...
+      - option_category: Secret of Evermore
+        option_name: difficulty
+        option_result: pupdunk_mystery
+        options:
+          Secret of Evermore:
+            fix_wings_glitch: false
+            difficulty: mystery
+  ```
+
+In this example if the `pupdunk` option is rolled then the difficulty values will be rolled again using the new options `normal`, `pupdunk_hard`, and `pupdunk_mystery`, and the exp modifier will be rerolled using new weights for 150 and 200. This allows for two more triggers that will only be used for the new `pupdunk_hard` and `pupdunk_mystery` options so that they will only be triggered on "pupdunk AND hard/mystery".
```
Comment (Ijwu) [inline WebHostLib/static/assets/tutorial/archipelago/triggers_en.md:135]: This looks duplicated.

Comment (alwaysintreble) [reply WebHostLib/static/assets/tutorial/archipelago/triggers_en.md:135]: it was i asked for permission first but going to credit him here as well

Comment (Ijwu) [reply WebHostLib/static/assets/tutorial/archipelago/triggers_en.md]: No, I mean it is a section of text directly below an identical section of text.

**Comment by Ijwu** [comment]
LGTM

**Pull request merged**